packages/service-integration/src/service_integration/_compose_spec_model_autogenerated.py

```diff
@@ -2,10 +2,11 @@
 #   filename:  https://raw.githubusercontent.com/compose-spec/compose-spec/master/schema/compose-spec.json
 #   timestamp: 2021-11-19T10:40:07+00:00
 
-from __future__ import annotations
+# type: ignore
```

Comment:
why is this at the beginning of the file?

Reply:
to ignore the entire file from mypy. Thisi s auto-generated